src/OSPSuite.Core/Domain/Populations/IndividualValues.cs

```diff
@@ -5,6 +5,8 @@ namespace OSPSuite.Core.Domain.Populations
 {
    public class IndividualValues
    {
+      public int? IndividualId { get; set; }
```

Comment:
Null=>id will be set automatically. This may not be required as I believe this is only used in pksim when crfeating a population but better be consistent.